lib/system/io.nim

```diff
@@ -488,6 +521,14 @@ const
     #"rt", "wt", "w+t", "r+t", "at"
     # we always use binary here as for Nim the OS line ending
     # should not be translated.
+  NoInheritFlag =
```

Comment:
Move this above ``FormatOpen`` and do the concats at compile-time in the ``FormatOpen`` array.